src/components/Books.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BookItem from './BookItem';
+import AddBook from './AddBook';
+
+const Books = () => (
+  <div>
+    <ul>
+      <BookItem />
+    </ul>
+    <AddBook />
+  </div>
```

Comment:
- [ ] As part of the requirements the `BookItem` component should Take the book data as a prop and renders the title and the author

example : 
``` Js


const Books = () => {
const book_list =[{title: test,author: test2,id: 1},{...},{...}]
return (
  <div>
    <ul>
    book_list .map(book => (
      <BookItem data={book} />
       )
    </ul>
    <AddBook />
  </div>
)};

export default Books;
```
**_🛑  and kindly make sure to update your `BookItem` component  to handle the received data through props 🙏_**

Reply:
Hi @ichala.

Kindly see the updated Books.js using Props.